spec/models/govbox_set_spec.rb

```diff
@@ -126,6 +126,20 @@
         end
       end
 
+      context 'when there are no job results' do
+        subject(:govbox_set_json) do
+          described_class.new('jobs', affiliate, geoip_info, highlighting_options).as_json
+        end
+
+        before do
+          affiliate.jobs_enabled = true
+        end
+
+        it 'returns no_job_results international text' do
+          expect(govbox_set_json[:jobs]).to eq('No job openings in your region match your search')
```

Comment:
`jobs` has enough with dealing with two different types already `nil` and `Array`. Let's not add `String` to it. Also the `React` already knows this. I don't think we need to return it here.